chemprop/cli/common.py

```diff
@@ -9,39 +10,40 @@
 
 
 def add_common_args(parser: ArgumentParser) -> ArgumentParser:
-    data_args = parser.add_argument_group("input data parsing args")
+    data_args = parser.add_argument_group("shared input data args")
```

Comment:
Very minor comment: should we capitalize this?

Reply:
I have a slight preference to capitalize the first word in the argument group names, but they were all lowercase to start with, so I didn't. If you prefer them to be capitalized, I am happy to do that.